gno.land/pkg/gnoland/app_test.go

```diff
@@ -66,13 +66,15 @@ func TestNewAppWithOptions(t *testing.T) {
 					},
 				},
 			},
-			Params: []Param{
-				{key: "foo", kind: "string", value: "hello"},
-				{key: "foo", kind: "int64", value: int64(-42)},
-				{key: "foo", kind: "uint64", value: uint64(1337)},
-				{key: "foo", kind: "bool", value: true},
-				{key: "foo", kind: "bytes", value: []byte{0x48, 0x69, 0x21}},
-			},
+			/*
```

Comment:
This is commented out since it conflicts with the alternative approach for the Params module in this PR. I believe we should avoid setting arbitrary parameter values in the genesis file without going through module validation.

We need to discuss it and agree on the approach.